packages/legend-application-studio/src/components/editor/edit-panel/uml-editor/AssociationEditor.tsx

```diff
@@ -113,6 +115,21 @@ const AssociationPropertyBasicEditor = observer(
       }
       return false;
     };
+    const isPropertyInvalid = (): boolean => {
+      const _association =
```

Comment:
Why are we refetching the assoication if its defined above. 
I think its okay to show this error even on dependency association since a reference to system should `NEVER` happen unless the 2 classes and assoication are all in system.
So for this check i would just say do a check that assoication is not in system and that the proeprty type is in system. if that is both true than its invalid.